packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -35,7 +35,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:2.6.1'
+    return 'defi/defichain:HEAD-f5373b2'
```

Comment:
https://github.com/DeFiCh/ain/pull/1102 has been merged

lets's wait for a new build into master then we can update this to point to it.